### ISSUE TITLE
target_type appears as diff

### ISSUE THREAD

**Issue by limitusus** [codenize-tools/applb#4]
Existing ALB shows the following diff these days:

```
I, [2017-11-09T12:09:00.514893 #69443]  INFO -- : Modify target group tg-test
I, [2017-11-09T12:09:00.532015 #69443]  INFO -- : <diff>
  :name=>"tg-test",
  :port=>80,
  :protocol=>"HTTP",
- :target_type=>"instance",
  :unhealthy_threshold_count=>3,
  :vpc_id=>"vpc-abcdefgh"}
```

`target_type` is not defined as DSL, so there is no workaround for now.

**Comment by wata-gh** [comment]
thanks for the issue!
I fixed it by #5

**Comment by limitusus** [comment]
Confirmed fix. Thanks!
